pandas/tests/reshape/test_merge.py

```diff
@@ -724,6 +724,63 @@ def test_indicator(self):
                           how='outer', indicator=True)
         assert_frame_equal(test5, hand_coded_result)
 
+    def test_validation(self):
```

Comment:
maybe just do some simple validation then (and leave them where they are). having completely duplicated tests is not that useful.